### PR TITLE
doc: add the step to install our library in readme

### PR DESCRIPTION
**Pull request by ArmandMeppa**

### CODE REVIEW

**Review by Marcjazz — COMMENTED** [review]
Will approve this PR once the source code on the main branch has completely integrated the `de.adorsys` domain

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+### Maven
+
+Add the following dependency to your `pom.xml` file:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>de.adorsys</groupId>
+    <artifactId>sd-jwt</artifactId>
+    <version>1.0-SNAPSHOT</version>
```
Comment (Marcjazz) [inline README.md:18]: Is the SNAPSHOT required ?

**Review by Marcjazz — COMMENTED** [review]
I tried importing it without success

File: README.md (hunk 1 above)
Comment (IngridPuppet) [reply README.md:18]: I suggest to add a placeholder encouraging consumers to look for for the latest version on Maven.

**Comment by IngridPuppet** [comment]
@ArmandMeppa Please attend this PR, and let us have it closed. We'll need another release with a bug fix on Maven Central.

**Pull request closed without merging**